subcommands/services/plakar-services.1

```diff
@@ -0,0 +1,70 @@
+.Dd July 8, 2025
+.Dt PLAKAR-SERVICES 1
+.Os
+.Sh NAME
+.Nm plakar-services
+.Nd Manage optional Plakar-connected services
+.Sh SYNOPSIS
+.Nm plakar services status Ar service\_name
+.Nm plakar services enable Ar service\_name
+.Nm plakar services disable Ar service\_name
+.Sh DESCRIPTION
+The
+.Nm plakar services
+command allows you to enable, disable, and inspect additional services that
+integrate with the
+.Nm plakar
+platform via
+.Xr plakar-login 1
+authentication.
+These services connect to the plakar.io infrastructure, and should only be
+enabled if you agree to transmit non-sensitive operational data to plakar.io.
+.Pp
+All subcommands require prior authentication via
+.Xr plakar-login 1 .
+.Pp
+At present, only the
+.Dq alerting
+service is available.
+When enabled, alerting will:
+.Bl -enum
+.It
+Send email notifications when operations fail.
+.It
+Expose the latest alerting reports in the Plakar UI (see
+.Xr plakar-ui 1
+).
+.El
+.Pp
+By default, all services are disabled.
+.Sh SUBCOMMANDS
+.Bl -tag -width Ds
+.It Ar status Ar service\_name
```

Comment:
do we really need to "escape" the underscores?

Reply:
no, removed